p4-16/psa/PSA.mdk

```diff
@@ -398,7 +400,17 @@ The function `platform_port_valid()` mentioned below takes a value of
 type `PortId_t`, returning `true` only when the value represents an
 output port for the implementation.  It is expected that for some PSA
 implementations there will be bit patterns for a value of type
-`PortId_t` that do not correspond to any port.
+`PortId_t` that do not correspond to any port.  This function returns
+true for `PORT_CPU`, but false for `PORT_RECIRCULATE`.
+`PORT_RECIRCULATE` is only intended as a value for `ingress_port` when
+ingress processing begins for a recirculated packet (see Table
+[#ingress-initial-values].  See section [#sec-after-egress] for how P4
+programs cause a packet to be recirculated.
```

Comment:
The additions above are worth reading and thinking about to see if they sound correct.

Reply:
Since it is used only in the pseudo-code, I think we're ok, even though we still consider PORT_RECIRCULATE a port, so it will give readers a pause.